posthog/temporal/tests/batch_exports/test_bigquery_batch_export_workflow.py

```diff
@@ -57,15 +49,32 @@ def assert_events_in_bigquery(client, table_id, dataset_id, events, bq_ingested_
         inserted_event = {k: json.loads(v) if k in json_columns and v is not None else v for k, v in row.items()}
         inserted_events.append(inserted_event)
 
+    # Reconstruct bq_ingested_timestamp in case we are faking dates.
```

Comment:
BigQuery exports include a `bq_ingested_timestamp` field to let the user know when the data was actually exported. This date is set at export time, so we don't know it before we run the test.

To counter this, I use `freezegun` to `freezetime` so that we can set during the test  what that date will be, and then assert it was properly set. However, `freezetime` will make it so instead of returning an instance of `dt.datetime` when obtaining an export time we return a `FakeDatetime`. This `FakeDatetime` is being compared to the `dt.datetime` returned by querying BigQuery. An equality check between `dt.datetime` and `FakeDatetime` fails, probably because [freezegun's `FakeDatetime`](https://github.com/spulec/freezegun/blob/4f4496380deefceead7bef23bccaca17c2bdecfa/freezegun/api.py#L351) doesn't override `__eq__` to allow this.

So, what we do here is manually initialize a `dt.datetime` instance from the datetime we are working with, in case that happens to be a `FakeDatetime`, so that we can compare them without issues.

The fact it's a `FakeDatetime` happens to be a test side-effect that we work around. It would be nice if `freezegun` just worked with `dt.datetime`. A better solution would, perhaps, be to try out  [`time-machine`](https://github.com/adamchainz/time-machine), but since we use `freezegun` already in the rest of PostHog, I didn't want to add another dependency to do the same.